tftp/tftp.go

```diff
@@ -6,16 +6,29 @@ import (
 	"os"
 	"time"
 
```

Comment:
just drop this newline then and should be all good... I thought we had something to catch this in CI already

Reply:
I think I just had a script to do this locally, but I've learned about gofumpt and iirc it takes care of this case.